package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "pretest": "npm run build",
     "test": "mocha --ui bdd --recursive ./dist/test --timeout 90000",
+    "tran": "mocha --ui bdd --recursive ./dist/test/model/transaction/Transaction.spec --timeout 90000",
```

Comment:
why?

Reply:
sorry, I was just testing and forgot to remove it :P it's faster because somehow in my console mocha couldn't filter this `./src/test/model/transaction/Transaction.spec`